### DISCUSSION TITLE
Convert folders to decks and subdecks?

### DISCUSSION THREAD

**Discussion by TfTHacker** [st3v3nmw/obsidian-spaced-repetition#144] [Q&A] [answered]
It isn't clear to me how to use this feature. Can I specify a folder as containing flash cards?

**Comment by st3v3nmw** [comment, marked as answer]
Not exactly, the plugin will automatically search the vault for folders with flashcards then display those.

**Comment by TfTHacker** [comment]
Oh, that is nice as well. It wasn't  clear to me from the docs.

By the way, the nested tag support is delightful, really helps with organizing.

**Reply by st3v3nmw** [reply]
Okay, I'll update the docs.
